Model/src/main/java/org/gusdb/wdk/model/user/dataset/event/repo/UserDatasetEventRepo.java

```diff
@@ -0,0 +1,458 @@
+package org.gusdb.wdk.model.user.dataset.event.repo;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Types;
+import java.time.LocalDateTime;
+import java.util.*;
+import javax.sql.DataSource;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gusdb.fgputil.db.runner.SQLRunner;
+import org.gusdb.wdk.model.WdkRuntimeException;
+import org.gusdb.wdk.model.user.dataset.UserDatasetType;
+import org.gusdb.wdk.model.user.dataset.event.model.UserDatasetEventStatus;
+import org.gusdb.wdk.model.user.dataset.event.model.EventRow;
+import org.gusdb.wdk.model.user.dataset.event.model.UserDatasetEventType;
+
+/**
+ * Repository for DB actions related to user dataset events.
+ * <p>
+ * UserDatasetEvent Table:
```

Comment:
@Foxcapades This is not in the schema:
https://github.com/VEuPathDB/ApiCommonData/blob/master/Load/lib/sql/apidbschema/createUserDatasetCoreTables.sql#L53

The columns should be defined in that file so that any new -rbld database is already right. 

How do we do an integration test, and then roll this change out? I'm now thinking:
- make a patch to all -inc databases that truncates the tables to wipe the QA data and recreates the schema with the new columns
- change one Jenkins job to use the ud-update branches and applying the patch to one QA database (if you like we can do this together and use mbio-inc, or just, like, use one of the databases that have stuff from QA Galaxy)
- apply the patch everywhere and merge the code
- wait for the release. It will then work out.

If you've already used a database to test this, could you make a PR to ApiCommonData with the schema you used?

Reply:
There already is a PR for the ApiCommonData tables: https://github.com/VEuPathDB/ApiCommonData/pull/7